tests/scripts/thread-cert/border_router/test_multi_ail.py

```diff
@@ -65,56 +63,66 @@ class TwoBorderRoutersOnTwoInfrastructures(thread_cert.TestCase):
         BR2: {
             'name': 'BR_2',
             'backbone_network_id': 1,
-            'allowlist': [BR1],
+            'allowlist': [BR1, BR3],
+            'is_otbr': True,
+            'version': '1.3',
+        },
+        BR3: {
+            'name': 'BR_3',
+            'backbone_network_id': 1,
+            'allowlist': [BR2],
             'is_otbr': True,
             'version': '1.3',
         }
     }
 
-    def test(self):
+    def test_multi_backbone_infra(self):
```

Comment:
Does this naming work well? I think other test cases are using `test()` as the name. I think we should follow the convention.

Reply:
this can works perfect and the unittest framework, by default,  is able to parse all the methods start with `test`, see https://docs.python.org/3/library/unittest.html#unittest.TestLoader.testMethodPrefix

I am not using `test()` here because I plan to add another more test method inside this test class.